src/CSS/CSS.css

```diff
@@ -36,8 +40,45 @@
     word-break: break-word;
 }
 
+.btn-hover:active{
+    border: none;
+}
+
+.btn-icon{
+    text-decoration: none;
+    padding: 1rem;
+}
+
+.btn-outline{
+    text-decoration: none;
+    padding: 1rem;
+    font-size: 2rem;
+}
+
+.btn-icon:hover{
+    transform: scale(0.98);
+    opacity: 0.9;
+}
+
+.btn-hover:hover{
+    background-color: var(--success-hover-color);
+}
+
+.btn-outline:hover{
+    background-color: var(--warning-hover-color);
+}
+
 .code-snippet{
 display: inline-block;
     background-color: gray;
     padding: 1rem;
```

Comment:
Use variables instead of values. Will help in management.

Reply:
okay done